docs/bills-matching.md

```diff
@@ -0,0 +1,139 @@
+# Bills/transactions matching algorithm
```

Comment:
I think it would be good to have it on docs.cozy.io and since cozy-banks documentation is not there, it will not be there (maybe cozy-banks documentation should be added to docs.cozy.io) ?

Reply:
Maybe we could add a link to the FAQ https://support.cozy.io/category/132-cozy-banks ?

Reply:
cozy-banks documentation should be added to docs.cozy.io. I will open a PR to organize banks doc correctly and another one to add banks doc to docs.cozy.io.